augur/ancestral.py

```diff
@@ -228,21 +232,40 @@ def register_parser(parent_subparsers):
 
     return parser
 
-def run(args):
-    # Validate arguments.
+def check_arg_combinations(args, is_vcf):
```

Comment:
```suggestion
def validate_arguments(args, is_vcf):
```

Suggestion: rename to match the [similar function in augur filter](https://github.com/nextstrain/augur/blob/a5297310e8b53ccdfff625ded889b89be98ed2a8/augur/filter/validate_arguments.py), to maintain consistency across subcommands.